src/phoenix/db/insertion/trace_annotation.py

```diff
@@ -36,10 +38,21 @@ class TraceAnnotationQueueInserter(
         Precursors.TraceAnnotation,
         Insertables.TraceAnnotation,
         models.TraceAnnotation,
+        TraceAnnotationDmlEvent,
     ],
     table=models.TraceAnnotation,
     unique_by=("name", "trace_rowid"),
 ):
+    async def _events(
+        self,
+        session: AsyncSession,
+        *insertions: Insertables.TraceAnnotation,
+    ) -> List[TraceAnnotationDmlEvent]:
+        records = [dict(as_kv(ins.row)) for ins in insertions]
+        stmt = self._insert_on_conflict(*records).returning(self.table.id)
+        ids = tuple([_ async for _ in await session.stream_scalars(stmt)])
+        return [TraceAnnotationDmlEvent(ids)]
```

Comment:
any chance we can pick a different name for this event? I was unfamiliar with DML as an acronym

Reply:
maybe just `MutationEvent`?

Reply:
fyi, sqlalchemy uses [DML](https://github.com/sqlalchemy/sqlalchemy/blob/3ac034057ce621379fb8e0926b851a903d2c7e0b/lib/sqlalchemy/sql/dml.py)

Reply:
> MutationEvent

This conflates with graphql mutations (which is not limited to databases).

Reply:
yeah I understand it's used, I'm just noting that I think it's probably more accessible to us and new engineers if we use names that don't require domain knowledge to recognize

Reply:
on the other hand, i think this is the domain knowledge that is _required_ for new engineers